change-viewpoint/README.md

```diff
@@ -0,0 +1,58 @@
+# Map Load Status
+This sample demonstrates different ways to change the ```ViewPoint``` of a map. 
+
+
+![changevp1](https://cloud.githubusercontent.com/assets/12448081/15730683/8962423a-2823-11e6-8893-1a963ba07c8d.png)
+
+Three methods are demonstrated to change the view point namely:
+* Geometry
+
+ Zoom to a real-world feature - Griffith Park by creating geometry from JSON file
+ 
+![changevp2](https://cloud.githubusercontent.com/assets/12448081/15730692/9bf3d30a-2823-11e6-8a1d-de959f062772.png)
+
+* Center and Scale 
+
+ Zoom to Waterloo
+
+![changevp3](https://cloud.githubusercontent.com/assets/12448081/15730697/a66ac8a2-2823-11e6-8726-c559c0287e00.png)
+
```

Comment:
Use tables if you want to use multiple screenshots, an example can be found [here](https://github.com/Esri/arcgis-runtime-samples-android/blob/quartz-beta/feature-layer-update-attributes/README.md)